packages/sdk/src/sync-agent/syncAgents.test.ts

```diff
@@ -215,4 +217,59 @@ describe('syncAgents.test.ts', () => {
             { timeoutMS: 10000 },
         )
     })
+
+    test('leave gdm', async () => {
```

Comment:
these tests are run in sequence with the same users, can you leave the existing gdm?